service/src/main/java/uk/gov/hmcts/reform/fpl/utils/AgeDisplayFormatHelper.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import java.time.Clock;
+import java.time.LocalDate;
+import java.time.Period;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class AgeDisplayFormatHelper {
+
+    private AgeDisplayFormatHelper() {
+    }
+
+    public static String formatAgeDisplay(final LocalDate dateOfBirth) {
+        checkNotNull(dateOfBirth, "Date of birth value is required");
+
+        LocalDate today = LocalDate.now(Clock.systemDefaultZone());
```

Comment:
Would say that using time here is probably more sensible, seeing as that is how we are doing it elsewhere, although that does mean that to autowire it in you would need to make this a component.